src/orm/models/pilots.js

```diff
@@ -77,6 +77,13 @@ const Pilot = db.define(
       validate: { isIn: [ ['meters', 'feet'] ] }
     },
 
+    distanceUnit: {
+      type: Sequelize.ENUM('meters', 'kilometers', 'miles'), // eslint-disable-line new-cap
+      allowNull: false,
+      defaultValue: 'kilometers',
+      validate: { isIn: [ ['meters', 'kilometers', 'miles'] ] }
+    },
+
```

Comment:
Nice, very nice! Thanks!

I see you even made a DB migration. Although I thought about distance as horizontal measurement unit. I don’t think that anyone is going to measure there triangles in meters, or XC flight distance in meters. And meters are just a smaller distance measurement unit in the same metric system.

So, I would leave only kilometers and miles, meters are excessive. Kilometers will be default, and miles will have a conversion value.

If you need small distances to show up nicely in the UI, it's better to format them in the UI.

Reply:
Sure - you are right, it was just that 'meters' came for free :-).